### PR TITLE
Models DB / Sequelize

### PR DESCRIPTION
**Pull request by SebaSatelier**
adding the following models:
- Location
- w_models
- zer0 to zer7

### CODE REVIEW

**Review by FacundoPucheta — DISMISSED** [review]

**Review by FacundoPucheta — DISMISSED** [review]

File: src/models/Location.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        },
+    latitude: {
+            type: DataTypes.STRING(10),
+            defaultValue: null,
+        },
+    longitude: {
+            type: DataTypes.STRING(10),
+            defaultValue: null,
+        },
+    pol: {
```
Comment (manduinca) [inline src/models/Location.js]: I think that `polygon` can be a more suitable name.

File: src/models/Location.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  // define model location
+sequelize.define('location', {
+    id: {
+            //primary key.
+            type: DataTypes.INTEGER,
+            primaryKey : true,
+            validate: {
+                notEmpty : true
+            }
+
```
Comment (manduinca) [inline src/models/Location.js]: Please, delete this empty line.

File: src/models/W_one.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+sequelize.define('w_one', {
+    id: {
+        //primary key, autoincrement
+        type: DataTypes.INTEGER,
+        primaryKey : true,
+        autoIncrement: true,
+        validate: {
+            notEmpty : true
+            }
+
```
Comment (manduinca) [inline src/models/W_one.js]: I think this line is unnecessary, right?

File: src/models/Zer0.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+const { DataTypes } = require('sequelize');
+
+module.exports = (sequelize) => {
+sequelize.define('zer0', {
+    id: {
+        //primary key
+        type: DataTypes.INTEGER.UNSIGNED,
+        primaryKey : true,
+        allowNull: false
+        },
```
Comment (manduinca) [inline src/models/Zer0.js]: Please, fix the indentation.

File: src/models/Zer0.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    Periodo: {
+        type: DataTypes.STRING(5),
+        defaultValue: null,
+        },
```
Comment (manduinca) [inline src/models/Zer0.js]: Please, fix the indentation.

File: src/models/Zer0.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    X: {
+        type: DataTypes.STRING(10),
+        defaultValue: null,
+        },
```
Comment (manduinca) [inline src/models/Zer0.js]: Please, fix the indentation.

File: src/models/Zer0.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    Y_y: {
+        type: DataTypes.STRING(30),
+        defaultValue: null,
+        },
```
Comment (manduinca) [inline src/models/Zer0.js]: Please, fix the indentation.

File: src/models/Zer0.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+        defaultValue: null,
+    },
+    Y_mc: {
+        type: DataTypes.STRING(30),
+        defaultValue: null,
+    },
+    Y_ab: {
+        type: DataTypes.STRING(30),
+        defaultValue: null,
+        },
```
Comment (manduinca) [inline src/models/Zer0.js]: Please, fix the indentation.

File: src/models/Zer0.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+module.exports = (sequelize) => {
+sequelize.define('zer0', {
+    id: {
+        //primary key
+        type: DataTypes.INTEGER.UNSIGNED,
+        primaryKey : true,
+        allowNull: false
+        },
+    Periodo: {
```
Comment (manduinca) [inline src/models/Zer0.js]: In english please, maybe `period`.

File: src/models/Zer0.js [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+    Y_bc: {
+        type: DataTypes.STRING(30),
+        defaultValue: null,
+        }},
+        {
+    timestamps: false,
+    indexes: [
+            {
+                name: 'id_index',
+                fields: ['id'], // Definir la columna 'id' como índice
```
Comment (manduinca) [inline src/models/Zer0.js]: I think the comment is unnecessary.

Comment (manduinca) [inline src/models/Zer0.js]: If `id` is a pk. Already is not `id` an index?

File: src/models/W_one.js [hunk 11]
```diff
@@ -0,0 +1,48 @@
+const { DataTypes } = require('sequelize');
+
+module.exports = (sequelize) => {
+sequelize.define('w_one', {
+    id: {
+        //primary key, autoincrement
```
Comment (manduinca) [inline src/models/W_one.js]: The comment is unnecessary, I think.

**Review by manduinca — CHANGES_REQUESTED** [review]
Please, check the comments. I put some questions. Another general question is: Why do we use different ways to write the primary key?

**Comment by SebaSatelier** [comment]
fixed indentation errors (prettier format),
change variable name pol and periodo to polygon and period,
change validate notEmpty to allowNull,
change indexes to Primary Key

**Review by manduinca — APPROVED** [review]
LGTM!

**Review by FacundoPucheta — APPROVED** [review]

**Pull request merged**